nn/transformer.py

```diff
@@ -11,7 +11,7 @@ class TransformerEncoderLayer(nn.Module):
   """
   Defines one layer of a standard transformer encoder
   """
-  def __init__(self, dim_model: int, num_heads: int, dim_ff: int = 2048, dropout: float = 0.1,
+  def __init__(self, dim_model: int, num_heads: int, dim_ff: int = 2048, dropout: float = 0.1, att_dropout: float = 0.1,
```

Comment:
Why do you use these defaults for `dropout` and `att_dropout`? Can you point to some recent published papers where they use the same, or some other references?

Reply:
I would recommend to not put any defaults at all. ASR/MT and TTS can all have very different reasonable values (usually between 0.1 and 0.5).